static/app/actionCreators/sessions.tsx

```diff
@@ -0,0 +1,64 @@
+import {Client} from 'sentry/api';
+import {getInterval} from 'sentry/components/charts/utils';
+import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
+import {DateString, Organization, SessionApiResponse} from 'sentry/types';
+import {defined} from 'sentry/utils';
+
+export type DoSessionsRequestOptions = {
+  field: string[];
+  orgSlug: Organization['slug'];
+  cursor?: string;
+  end?: DateString;
+  environment?: Readonly<string[]>;
+  groupBy?: string[];
+  interval?: string;
+  orderBy?: string;
+  project?: Readonly<number[]>;
+  query?: string;
+  start?: DateString;
+  statsPeriod?: string | null;
+  statsPeriodEnd?: string;
+  statsPeriodStart?: string;
+};
+
+export const doSessionsRequest = (
```

Comment:
should we remove the file https://github.com/getsentry/sentry/blob/09dd1f6b2e7b1a839ad0eac8a038a0f4871e9dec/static/app/actionCreators/metrics.tsx + test?